src/box/wal.h

```diff
@@ -68,7 +68,7 @@ enum {
 	 * applied from WAL. It allows not to block the event
 	 * loop for the whole recovery stage.
 	 */
-	WAL_ROWS_PER_YIELD = 1 << 15,
+	WAL_ROWS_PER_YIELD = 1 << 5,
```

Comment:
Why?

Reply:
`WAL_ROWS_PER_YIELD = 1 << 5`  leave only for the test by entering a new errinj.